test/server/server.test.ts

```diff
@@ -0,0 +1,219 @@
+import {FastifyInstance} from 'fastify';
+import {Server} from 'http';
+import {
+  Catalog,
+  Dataset,
+  IRI,
+  Organization,
+  SparqlDistribution,
+} from '@netwerk-digitaal-erfgoed/network-of-terms-catalog';
+import {server} from '../../src/server/server';
+import {gql} from 'mercurius-codegen';
+import {setup, teardown} from 'jest-dev-server';
+
+let httpServer: FastifyInstance<Server>;
+const catalog = new Catalog([
+  new Dataset(
+    new IRI('http://vocab.getty.edu/aat'),
```

Comment:
Add support for multiple URI prefixes?

Reply:
This is the Dataset URI itself; not yet the URI prefix functionality, for which see #251.

Reply:
Ah, I see! Perfect!